frontend/src/components/commons/Tab.jsx

```diff
@@ -24,23 +35,46 @@ const styles = {
 class Tab extends Component {
   static propTypes = {
     tabName: PropTypes.string.isRequired,
-    selected: PropTypes.bool.isRequired
+    selected: PropTypes.bool.isRequired,
+    // use this prop to disable the tab
+    disabled: PropTypes.bool
   };
 
   render() {
     return (
       <span>
-        {this.props.selected === false && (
-          <li role="menuitem" style={styles.li}>
-            <button style={styles.button} className="side-navigation-button">
-              {this.props.tabName}
-            </button>
-          </li>
+        {!this.props.disabled && (
+          <span>
+            {this.props.selected === false && (
```

Comment:
You should use `this.props.selected` and `!this.props.selected` instead of comparing to a boolean.

Reply:
Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/241/commits/6a209ca1263b7ced2d2358f642d93881e762b92c